themes/mlibrary_new/exhibit-builder/exhibits/browse.php

```diff
@@ -13,9 +13,21 @@
             )
        );
 
+
 ?>
+  <div class="col-xs-9">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item"><?php echo link_to_home_page(__('Home')); ?></li>
+        <li class="breadcrumb-item active">Browse all exhibits</li>
+      </ol>
+  </div>
+
+
+  <?php
+    $html = mlibrary_display_popular_tags();
```

Comment:
Probably just `echo mlibrary_display_popular_tags();` here.

Reply:
I still work on this file, so I need this variable

Reply:
Changes that you make for debugging code shouldn't get included into the main branch.  I advocate for taking it out of the commit before making a PR.

Reply:
The $html is not for debugging, I will use this variable, I am still working on this page.